elastic/tests/test_elastic.py

```diff
@@ -84,6 +84,63 @@ def test__get_urls(instance, url_fix):
     assert slm_url == ('/_slm/policy' if instance.get('slm_stats') is True else None)
 
 
+@pytest.mark.integration
+def test_custom_queries_valid_metrics(dd_environment, dd_run_check, instance, aggregator):
+    custom_queries = [
+        {
+            'endpoint': '/_nodes',
+            'metrics': [
+                {
+                    'datadog_metric_name': 'elasticsearch.custom.metric',
+                    'es_metric_name': '_nodes.total',
+                    'type': 'monotonic_count',
+                    'tags': ['custom_tag:1'],
+                },
+            ],
+        },
+        {
+            'endpoint': '/_nodes',
+            'metrics': [
+                {
+                    'datadog_metric_name': 'elasticsearch.custom.metric2',
+                    'es_metric_name': '_nodes.total',
+                    'tags': ['custom_tag:1'],
+                },
+            ],
+        },
+    ]
+    instance = deepcopy(instance)
+    instance['custom_queries'] = custom_queries
+    check = ESCheck('elastic', {}, instances=[instance])
+    dd_run_check(check)
+
+    aggregator.assert_metric('elasticsearch.custom.metric')
```

Comment:
```suggestion
    aggregator.assert_metric('elasticsearch.custom.metric', type=)
```
Can you also assert type and tags here?